Sources/SATSCore/Components/DatePickerView/Subviews/DatePickerView-DayView.swift

```diff
@@ -0,0 +1,23 @@
+import SwiftUI
+
+extension DatePickerView {
+    struct DayView: View {
+        let day: String
+        let textColor: Color
+        let backgroundColor: Color
+
+        var body: some View {
+            VStack(spacing: 10) {
+                Text(day)
+            }
```

Comment:
Asking for a friend about why this is a `VStack` if it only contains a single child view. Or is it more a case of “any stack will do” since all you really want is just a container/wrapper for the shape and color?

Reply:
You're absolutely right, @sindrenm ! Nicely spotted. Guess there was some refactor blindness here. I don't need this.